spec/context/dimension.fmf

```diff
@@ -0,0 +1,50 @@
+summary: Supported context dimensions
+
+story:
+    As a tester I want to clearly specify dimensions, such as
+    'distro' or 'trigger', so that I can well describe the
+    context in which a test is running.
+
+description: |
+    The following dimensions are supported:
+
+    product
+        The target product name for which the application is
+        intended (fedora, fedora-33, centos, centos-8,
+        centos-84, rhel, rhdts, rhscl).
+
+    distro
+        The operating system distribution on which the application
+        runs (fedora, fedora-33, centos, centos-8, centos-8.4,
+        rhel).
+
+    variant
+        The distribution variant (Client, Desktop, Server
+        Workstation).
+
+    arch
+        The guest architecture (aarch64, armhfp, i386, ppc64,
+        ppc64le, s390x, x86_64).
+
+    collection
+        The Red Hat Software Collection name (python27,
+        rh-python34, rh-mariadb100, httpd24).
+
+    component
+        Name of the relevant component, should match the source
+        package name (bash, php, httpd).
```

Comment:
Does this trickle somehow to the individual (beakerlib) test? Is some ENV set?

Reply:
No, but you can modify environment variables based on the context:

    adjust:
        environment:
            key: val
        when: distro == fedora